.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    
+
+env:
+  CARGO_TERM_COLOR: always
```

Comment:
This is likely not necessary.

Reply:
yes probably, it's what github has in its rust templates, so I left it in.